docs/versions/content-patterns.md

```diff
@@ -0,0 +1,313 @@
+---
+navigation_title: "Content patterns"
+applies:
+  stack: all
+  serverless: all
+  hosted: all
+  eck: all
+  ece: all
+---
+# Version content patterns
+
+Depending on what you're trying to communicate, you can use the following patterns to represent version and deployment type differences in your docs.
+
+Choose from:
+% this should become a quick-ref table
+
+* [Page-level `applies` tags](#page-level-applies-tags)
+* [Section/heading-level `applies` tags](#sectionheading-level-applies-tags)
+* [Inline `applies` tags](#inline-applies-tags) (currently just prose)
+* [Tabs](#tabs)
+* [Sibling bullets](#sibling-bullets)
+* [Callouts](#callouts)
+* [Prose (inline)](#prose-inline)
+* [Prose (explanatory paragraphs and sections)](#prose-explanatory-paragraphs-and-sections)
+* [Sibling pages](#sibling-pages)
+
+## Page-level `applies` tags
+
+*see [`applies`](/syntax/applies.md)*
+
+**Use case:** Provide signals that a page applies to the reader
+
+**Number to use:** Minimum to add clarity. See [basic concepts and principles](/versions/index.md#basic-concepts-and-principles). 
+
+**Approach:**
+
+Add tags for all **versioning facets** relevant to the page.
+
+See **Versions and lifecycle states** to learn when to specify versions in these tags.
+
+### Example
+[Manage your Cloud organization](https://docs-v3-preview.elastic.dev/elastic/docs-content/tree/main/deploy-manage/cloud-organization.html)
+
+### Section/heading-level `applies` tags
+:::{applies}
+:ece: all
+:hosted: all
+:eck: all
+:stack: all
+:::
+
+*see [`applies`](/syntax/applies.md#sections)*
+
+**Use case:** Provide signals about a section’s scope so a user can choose to read or skip it as needed
+
+**Number to use:** Minimum to add clarity. See [basic concepts and principles](/versions/index.md#basic-concepts-and-principles).  
+
+**When to use:** When the section-level scope differs from the page-level scope
+
+**Example**
+See above
+
+## Inline `applies` tags
+
+::::{warning}
+This feature doesn't currently work - we're working around it by using prose.
+::::
+
+**Use case:** When features in a **list of features** are exclusive to a specific context, or were introduced in a specific version
+
+**Number to use:** Three max. If the number of tags is longer than that, consider: 
+
+*  Breaking up the list by facet
+*  Using labels that don't have version / lifecycle information and deferring that information to the page or section for the feature
+  
+Currently, we don't have inline `applies` tags. Instead, append the information to the bullet item in bolded square brackets `**[]**`. Add all information within a single set of square brackets. The brackets should appear after any final punctuation. 
```

Comment:
> Currently, we don't have inline `applies` tags. Instead, append the information to the bullet item in bolded square brackets `**[]**`. 

Is it worth keeping this section or should we just remove it  and recommend that folks refer to `versions/content-patterns.html#prose-inline` instead. This seems like it might confuse people into using a format that doesn't seem like we're going to support.

Reply:
I can rename the section and tweak a bit